docs/source/topics/configfile.rst

```diff
@@ -45,6 +45,12 @@ be checked.
   will also be the URL prefix for your API
   (``https://endpoint/prefix/your-api``).
 
+* ``minimum_compression_size`` - An integer value that indicates
+  the minimum compression size to apply to the API gateway. If
+  this key is specified in both a stage specific config option
+  as well as a top level key, the stage specific key will
+  override the top level key for the given stage.
```

Comment:
It might be worth linking the service docs for this as well (e.g. https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-gzip-compression-decompression.html)